### DISCUSSION TITLE
PyTorch 1.11 + Torchvision 0.12 + TorchData updates

### DISCUSSION THREAD

**Discussion by mrdbourke** [mrdbourke/pytorch-deep-learning#43] [Announcements]
PyTorch released a few updates today.

The main ones being PyTorch version 1.11, Torchvision 0.12 + TorchData.

There were a few more but these ones will be the main updates concerning the course.

They don't change too much of the materials as of now (10 March 2022) but I'm leaving them here to:

1. Be aware of them
2. Refer back to them if needed for future updates to the course

For now, the course minimum version of PyTorch remains at 1.10.0.

* TorchData (new way to create datasets in PyTorch, it breakdowns `DataLoader`/`Dataset` into different parts) update blog post - https://pytorch.org/blog/pytorch-1.11-released/ 
* Other information on `torchvision` 0.12 and `torchtext` and `torchaudio` and `TorchRec` (PyTorch for recommendation systems) updates - https://pytorch.org/blog/pytorch-1.11-new-library-releases/

**Comment by TamirFriedman-RecoLabs** [comment]
Hi @mrdbourke :) do you cover TorchRec in the course? thnx
